boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -124,3 +114,46 @@ yaml-validate: python-venv
 .PHONY: olm-deploy-yaml-validate
 olm-deploy-yaml-validate: python-venv
 	${PYTHON} ${CONVENTION_DIR}/validate-yaml.py $(shell git ls-files 'deploy/*.yaml' 'deploy/*.yml')
+
+######################
```

Comment:
Should we separate these into a `prow.mk` to reduce file bloat?

Reply:
I actually talked about that with @jharrington22 before pulling the trigger on this PR. He preferred to keep it monolithic for searchability. I'd be happy to go either way.

With the nexus makefile include, it'd be trivial to move later if we feel like it.